File: backend/src/main/java/com/ftn/activityapp/dto/planner/PlannerSuggestionDto.java
package com.ftn.activityapp.dto.planner;

import com.ftn.activityapp.enums.PlannerSuggestionType;
import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PlannerSuggestionDto {
    private String title;
    private String description;
    private Integer suggestedDurationMinutes;
    private PlannerSuggestionType suggestionType;
    private String recommendedPartOfDay;
    private String taskType;
    private String source;
}

File: mobile-app/app/src/main/java/com/example/activityapp/data/remote/dto/planner/PlannerSuggestionDto.kt
package com.example.activityapp.data.remote.dto.planner

data class PlannerSuggestionDto(
    val title: String,
    val description: String,
    val suggestedDurationMinutes: Int,
    val suggestionType: String,
    val recommendedPartOfDay: String,
    val taskType: String,
    val source: String
)